docs/source/changelog.rst

```diff
@@ -8,6 +8,7 @@ Development version
 0.8.1 / 2022-10-04
 ------------------
 
+- Extend OARCluster implementation to let OAR take into account the memory parameter (:pr:`598`, :pr:`595`)
```

Comment:
This point should go into the development-version section above. The 0.8.1 is already fixed / released.

Reply:
oh sorry about my misunderstanding.. Do you already have in mind the version number / date for the next release? otherwise, I will just remove this PR.